simulation/halsim_ds_socket/src/main/native/cpp/main.cpp

```diff
@@ -124,33 +124,41 @@ static void SetupUdp(wpi::uv::Loop& loop) {
     });
   });
   simLoopTimer->Start(Timer::Time{100}, Timer::Time{100});
-
+  int timeoutMs = 100;
+  auto envTimeout = std::getenv("DS_TIMEOUT_MS");
+  if (envTimeout) {
+    timeoutMs = std::stoi(envTimeout);
+  }
+  auto autoDisableTimer = Timer::Create(loop);
+  autoDisableTimer->timeout.connect([] { HALSIM_SetDriverStationEnabled(0); });
+  autoDisableTimer->Start(Timer::Time(timeoutMs));
```

Comment:
Remove this start call--we want to start this timer on the first received packet.